### ISSUE TITLE
Add card hover animation

### ISSUE THREAD

**Issue by Clashsoft** [fujaba/fulibWorkflows#12]
It should look like I am trying to pick a card from the wall and debating whether to keep it or throw it in the trash. Translated into CSS:

```css
    .card {
        transition: all .5s;
        transform-style: preserve-3d;
        transform-origin: top;
        cursor: grab;
    }
    .card:hover {
        z-index: 1;
        transform: perspective(1000px) translateY(20px) translateZ(100px) rotateX(15deg);
        box-shadow: 0 1rem 3rem rgba(0,0,0,0.5) !important;
    }
```

Yes the grab cursor needs to be there

<img width="823" alt="Screenshot 2022-03-07 at 23 26 17" src="https://user-images.githubusercontent.com/4145923/157128328-4beb907a-e61f-41bd-beba-1d3802ba687d.png">

**Comment by Kuenssi** [comment]
Will be added to fulibWorkflows in the next release

**Comment by Kuenssi** [comment]
Released in v0.4.5
